fabric-rendering-v1/src/main/java/net/fabricmc/fabric/api/client/rendering/v1/BuiltinItemRendererRegistry.java

```diff
@@ -76,6 +78,7 @@
 	 * @throws IllegalArgumentException if the item already has a registered renderer
 	 * @throws NullPointerException if either the item or the renderer is null
 	 */
+	@Contract("null, _ -> fail; _, null -> fail")
```

Comment:
`@NotNull` on `ItemConvertible` and `DynamicItemRenderer` might be better than a `@Contract` annotation.